### PR TITLE
remove str, replace with {}.format

### PR DESCRIPTION
**Pull request by YourLocalBlake**

### CODE REVIEW

File: src/koala/__init__.py [hunk 1]
```diff
@@ -2313,7 +2312,7 @@ def RSS_map(
 
         cbar = plt.colorbar()
         plt.clim(np.nanmin(variable[list_spectra]), np.nanmax(variable[list_spectra]))
-        cbar.set_label(str(color_bar_text), rotation=90, labelpad=40)
+        cbar.set_label("{}".format(color_bar_text), rotation=90, labelpad=40)
```
Comment (aragilar) [inline src/koala/__init__.py]: Isn't color_bar_text on its own fine?

File: src/koala/__init__.py [hunk 2]
```diff
@@ -2358,7 +2357,7 @@ def RSS_image(
 
         # plt.colorbar()
         cbar = plt.colorbar()
-        cbar.set_label(str(color_bar_text), rotation=90, labelpad=labelpad)
+        cbar.set_label("{}".format(color_bar_text), rotation=90, labelpad=labelpad)
```
Comment (aragilar) [inline src/koala/__init__.py]: Same here

File: src/koala/__init__.py [hunk 3]
```diff
@@ -3593,7 +3592,7 @@ def __init__(
             plt.title(
                 self.object
```
Comment (aragilar) [inline src/koala/__init__.py]: Make this whole thing a string with format

File: src/koala/__main__.py [hunk 4]
```diff
@@ -56,13 +55,13 @@
 
     pk = (
```
Comment (aragilar) [inline src/koala/__main__.py]: Same here

File: src/koala/__init__.py (hunk 1 above)
Comment (YourLocalBlake) [reply src/koala/__init__.py]: yeah it is

File: src/koala/__init__.py (hunk 3 above)
Comment (YourLocalBlake) [reply src/koala/__init__.py]: This have been reserved for later.

File: src/koala/__main__.py (hunk 4 above)
Comment (YourLocalBlake) [reply src/koala/__main__.py]: Left for later.

**Comment by YourLocalBlake** [comment]
This should have been fixed in #102

**Comment by YourLocalBlake** [comment]
bors try

**Comment by YourLocalBlake** [comment]
with #103 completed, hopefully it should work now

**Comment by YourLocalBlake** [comment]
bors try

**Comment by YourLocalBlake** [comment]
3rd time is the charm? #102 #103, #104

**Comment by YourLocalBlake** [comment]
bors try

**Pull request merged**